
# General
safe_delay: 0.5
gravity_norm: 9.831
fixed_lag: 30
hot_start_delay: 3

crs_source: "EPSG:4326"
crs_target: "EPSG:6052"

# Prior covariance
initial_acc_bias_sigma: 0.5
initial_gyro_bias_sigma: 0.01

# Lever arm stuff   
lever_norm_threshold: 25
lever_norm_sigma: 0.1
lever_altitude_sigma: 1

# GNSS
gnss/timeout_interval: 5.0
gnss/suspension_interval: 10.0
gnss/sigma_xy: 2.0

# IMU
imu_timeout_interval: 1.1
imu_attitude_sigma: 0.1

accel_noise_sigma: 1.0e-3
gyro_noise_sigma: 1.0e-4
accel_bias_rw_sigma: 1.0e-4
gyro_bias_rw_sigma: 1.0e-5

# Surface estimation
surface_estimation/buffer_size: 2.0 # Allocate memory for at least buffer_size seconds worth of points
surface_estimation/lidar_point_interval: 5.0e-6   # Interval between each scan from lidar (put half if in dual-return mode)
surface_estimation/lidar_min_intensity: 0.0
surface_estimation/lidar_min_dist: 15
surface_estimation/lidar_max_dist: 100.0

surface_estimation/ransac_threshold: 0.5
surface_estimation/ransac_sample_size: 10
surface_estimation/ransac_iterations: 5
surface_estimation/ransac_inlier_count: 25
surface_estimation/ransac_frame_size: 0.1

surface_estimation/sigma_altitude: 1.0
surface_estimation/sigma_attitude: 0.2